exporter/importables.go

```diff
@@ -946,55 +939,8 @@ var resourcesMap map[string]importable = map[string]importable{
 			{Path: "source", File: true},
 		},
 	},
-	"databricks_sql_query": {
```

Comment:
for visibility: as discussed, I believe that we still need a functionality to export all queries, even those that aren't referenced from the dashboards - it would be useful for migrations.

Reply:
yep. one for following commits will bring back listing and introduce it into it's own "service". We need to start working on PRD for this :D